airflow/www/static/js/dag/details/dagRun/index.tsx

```diff
@@ -23,7 +23,7 @@ import {
   Box,
   Button,
   Link,
-  Divider,
+  Divider, Table, Tbody, Tr, Td, Heading,
```

Comment:
I don't know how the linting didn't catch this but usually we keep all the imports on separate lines.

```suggestion
  Divider,
  Table,
  Tbody,
  Tr,
  Td,
  Heading,
```